src/styles/blocks/nav.scss

```
@@ -0,0 +1,79 @@
+.nav {
+  margin: 0 50px 0 0;
+
+  &__list {
+    margin: 0;
+    padding: 0;
+
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+
+    list-style-type: none;
+  }
+
+  &__link {
+    height: 60px;
+    margin: 0 10px;
+
+    position: relative;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    color: $color-main-accent;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: 12px;
+
+    transition-duration: $transition-time;
+
+    &:link {
+      color: $color-main-accent;
+    }
+
+    &:visited {
+      color: $color-main-accent;
+    }
```

You can combine selectors
```suggestion
    &:link,
    &:visited {
      color: $color-main-accent;
    }
```